### PR TITLE
Adds Picket Signs

### PR DESCRIPTION
**Pull request by AnturK**
Tablecrafted from metal rod and two cardboard sheets. Urgently needs non-shit inhands. Basic icon too i guess.

scrnshot1

scrn2

### CODE REVIEW

File: code/game/objects/items/weapons/signs.dm [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	if(istype(W, /obj/item/weapon/pen) || istype(W, /obj/item/toy/crayon))
+		var/txt = stripped_input(user, "What would you like to write on the sign?", "Sign Label", null , 30)
+		if(txt)
+			label = txt
+			src.name = "[label] sign"
+			desc =	"It reads: [label]"
+	..()
+
+/obj/item/weapon/picket_sign/attack_self(mob/living/carbon/human/user)
+	if(label!="")
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/signs.dm]: ```if(label)```

File: code/game/objects/items/weapons/signs.dm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		user.visible_message("<span class='warning'>[user] waves around \the \"[label]\" sign.</span>")
+	else:
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/signs.dm]: nice colon, don't need it, throw it in the trash.

File: code/game/objects/items/weapons/signs.dm [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/obj/item/weapon/picket_sign
+	icon_state = "picket"
+	name = "blank picket sign"
+	desc = "It reads:"
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/signs.dm]: Could probably start off as "It is blank."

File: code/game/objects/items/weapons/signs.dm (hunk 2 above)
Comment (AnturK) [inline code/game/objects/items/weapons/signs.dm]: Blargf that's what i get for working on two thing in diffrent langs at the same time

Comment (RemieRichards) [inline code/game/objects/items/weapons/signs.dm]: Learn to separate the two mentally.

File: code/game/objects/items/weapons/signs.dm (hunk 1 above)
Comment (AnturK) [inline code/game/objects/items/weapons/signs.dm]: Sanitize does not remove leading whitespace ? I'll change it i guess

Comment (RemieRichards) [inline code/game/objects/items/weapons/signs.dm]: Leading whitespace? What? Your check here simply reads "If label is not empty" which is the same as ```if(label)```, which is just cleaner.

Comment (AnturK) [inline code/game/objects/items/weapons/signs.dm]: Ah thought you had issue with ___ signs, sure will change that

**Comment by WJohn** [comment]
The codersprites are real. Oh well it's only for a day anyway.

**Comment by Chocobro** [comment]
what a cool idea!

**Comment by Iamgoofball** [comment]
@WJohn this seems like something that could stick around

**Comment by Chocobro** [comment]
i'm a fan of the regular icon, but i'll fuck around with the inhands for you now.

**Comment by Chocobro** [comment]
here you go! used the colours of the regular icon to give it a bit more connectivity.
picketsigns
http://turd.church/ss13/shit-i-made/misc/picketsigns.dmi

**Comment by AnturK** [comment]
Neat. Will put some screenshots with these in a moment.

**Pull request merged**